packages/shipjs-lib/src/lib/config/defaultConfig.js

```diff
@@ -185,5 +185,6 @@ export default {
   },
   releases: {
     assetsToUpload: [],
+    extractChangelog: undefined, // ({ version, dir }) => `some specific changelog to that version`,
```

Comment:
I think this can go under `releases` section, because it's used only for creating GitHub releases, and for updating changelog.
What do you think?

Reply:
That's make sense!